docs/en/README.en.md

```diff
@@ -97,23 +97,9 @@ sh ./run.sh
 ## Preparation of other Pre-models
 RVC requires other pre-models to infer and train.
 
-You need to download them from our [Huggingface space](https://huggingface.co/lj1995/VoiceConversionWebUI/tree/main/).
```

Comment:
Well, you can keep this instruction if someone would like to download the files themselves. Just add the python instruction at the top.

Reply:
Check.